docs/ui/components/Sidebar/SidebarCollapsible.tsx

```diff
@@ -60,16 +61,16 @@ export function SidebarCollapsible(props: Props) {
 
   return (
     <>
-      <a css={titleStyle} onClick={toggleIsOpen}>
+      <ButtonBase css={titleStyle} aria-expanded={isOpen ? 'true' : 'false'} onClick={toggleIsOpen}>
         <div css={chevronContainerStyle}>
```

Comment:
I recommend adding `aria-hidden="true"` here. This element only adds visual information which is present via `aria-expanded` for assistive technology. At the moment, this icon has a `<title>` element inside, which gives the `svg` an assistive label, but this title is actually a bad title.

A collapsed section header announces something similar to:

```text
Chevron-down-icon graphic Configuration files button collapsed
```

An expanded one announces:

```text
Chevron-down-icon graphic Configuration files button expanded
```

This is because the same icon is used and rotated, so for AT users, it's just noise. Adding `aria-hidden="true"` to the icon or its parent changes this in:

```text
Configuration files button collapsed
```